.ci/Jenkinsfile

```diff
@@ -38,9 +35,6 @@ pipeline {
         deleteDir()
         gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
         stash(allowEmpty: true, name: 'source', useDefaultExcludes: false)
-        dir("${BASE_DIR}") {
-          setEnvVar('DOCKER_CHANGES', isGitRegionMatch(patterns: [ '^e2e/Dockerfile.jenkins' ], shouldMatchAll: false).toString())
```

Comment:
Oh, this reminded me of this [ticket]( #349) removing the Dockerfile for Jenkins, thanks! I can do it as a follow-up.